src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -753,11 +753,49 @@ def load_arguments(self, _):
     with self.argument_context('functionapp config appsettings') as c:
         c.argument('slot_settings', nargs='+', help="space-separated slot app settings in a format of `<name>=<value>`")
 
+    with self.argument_context('logicapp') as c:
+        c.argument('name', arg_type=logicapp_name_arg_type)
+
+    with self.argument_context('logicapp create') as c:
+        # Dynamic SKU is no longer supported for logicapp standard
+        c.argument('consumption_plan_location', options_list=['--consumption-plan-location', '-c'],
+                   help="Geographic location where {} app will be hosted. Use `az {} list-consumption-locations` to view available locations.".format(app_type, scope),
+                   deprecate_info=c.deprecate(expiration='2.41.0'))
+        c.argument('os_type', arg_type=get_enum_type(OS_TYPES), help="Set the OS type for the app to be created.",
+                   deprecate_info=c.deprecate(expiration='2.41.0'))
```

Comment:
> deprecate_info=c.deprecate(expiration='2.41.0')

May I ask why do you add these new parameters that will be deprecated in `2.41.0`?

Reply:
If these two parameters are soon deprecated, is it unnecessary to add these two parameters?

Reply:
@zhoxing-ms - the logicapp create is an existing command & we didn't want to remove these. The LA team confirmed the findings from Harroon's testing, that these are not supported.  Given we moved this release to Ignite time-frame now, can we just remove them & call this a breaking change?

Reply:
@panchagnula Got it, thanks~ We can merge and release this PR in this sprint, so that before making the breaking change in the Ignite sprint, we can prompt the deprecated information in advance to remind users to migrate their usage

Reply:
@haroonf @panchagnula Could you please submit a PR to remove these two deprecated parameters in time?

Reply:
> @haroonf @panchagnula Could you please submit a PR in the breaking change window _(Ignite sprint)_ to remove these two deprecated parameters in time?

Yes, will be sure to do that